linodecli/baked/colors.py

```diff
@@ -1,11 +1,22 @@
 """
-Applies shell color escapes for pretty printing
+Applies shell color escapes for pretty printing.
```

Comment:
This is unrelated to this change, but we should move to Rich tags for value rendering/colorizing in the future 👀 
I'll create a ticket for it when I get a chance